styles/components/table/_tableHeader.scss

```diff
@@ -2,6 +2,7 @@ th.table-header {
     border: 0;
     background-color: $color-white;
     padding: rem(15);
+    vertical-align: top;
```

Comment:
These change the default styles for anyone using this table and will need to be mentioned in the release notes.